source/dem/find_cell_neighbors.cc

```diff
@@ -111,5 +111,56 @@ FindCellNeighbors<dim>::find_cell_neighbors(
     }
 }
 
+// This function finds the full neighbor list (with repetition) of all the
+// active cells in the triangulation
```

Comment:
Just a question, why is this one needed? Especially with the repetition?
Maybe add a line that explains why this is needed and where.

Reply:
Added.